tests/transactional/enable_selinux.pm

```diff
@@ -25,14 +26,26 @@ sub run {
     systemctl 'is-enabled auditd';
 
     # install and enable SELinux if not done by default
-    if (script_run('test -d /sys/fs/selinux && test -e /etc/selinux/config') != 0) {
-        assert_script_run('transactional-update setup-selinux');
+    if (script_run('test -d /sys/fs/selinux && test -e /etc/selinux/config && getenforce | grep -i "enforcing" >/dev/null') != 0) {
```

Comment:
why do you add this command `getenforce | grep -i "enforcing"` ?

Reply:
5.2 is enabled in `permissive` mode by default. Why forcing it to re-enable it?

Reply:
Because on SLEM 5.1 SELinux is installed in permissive mode and we need to enable it in enforcing mode.

Reply:
No.
- 5.1 is not enabled by default. And enabling it results in enforcing mode.
- 5.2 and 5.3 enabled in permissive mode. `trup_call selinux` will go to enforcing -> bsc#1211058
- 5.4 enabled in enforcing mode.

Reply:
We need to discuss this. I see that we have different opinions on what should be achieved with this test run.

Reply:
We agreed that if SELinux is enabled, then the `enable_selinux` test should set it to `enforcing`.